examples/upgradeable-contracts/forward-calls/lib.rs

```diff
@@ -33,6 +33,30 @@ pub mod proxy {
         admin: AccountId,
     }
 
+    /// The trait provides information about `Proxy`
```

Comment:
What is the reason why you added this trait btw.?

Reply:
I added it to test the behavior of the wildcard in case of the trait is implemented before the inherent section.

But it was before I added the UI test=) So now it is not so actual but we can keep it to verify that the behavior is right during the e2e test.